inspirehep/config.py

```diff
@@ -164,6 +164,7 @@
 
 RECORD_EDITOR_INDEX_TEMPLATE = 'inspirehep_theme/invenio_record_editor/index.html'
 RECORD_EDITOR_PREVIEW_TEMPLATE_FUNCTION = get_detailed_template_from_record
+RECORD_EDITOR_FILE_UPLOAD_FOLDER = 'inspirehep/modules/editor/temp'
```

Comment:
What was the idea behind making this configurable?  Not enough with using https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp ?

Reply:
We were not sure if we wanted a local temporary file (mkstemp) or one on eos, as if not using eos it requires the sessions to be sticky in the load balancer.